src/frontend/src/pages/ChangeRequestDetailPage/StageGateDetails.tsx

```diff
@@ -3,27 +3,18 @@
  * See the LICENSE file in the repository root folder for details.
  */
 
-import { Grid, Typography } from '@mui/material';
 import { StageGateChangeRequest } from 'shared';
 import { booleanPipe } from '../../utils/pipes';
-import PageBlock from '../../layouts/PageBlock';
+import InfoBlock from '../../layouts/InfoBlock';
+import { Typography } from '@mui/material';
 
 interface StageGateDetailsProps {
   cr: StageGateChangeRequest;
 }
 
 const StageGateDetails: React.FC<StageGateDetailsProps> = ({ cr }) => {
   return (
-    <PageBlock title={'Stage Gate Change Request Details'}>
-      <Grid container spacing={1}>
-        <Grid item xs={3}>
-          <Typography sx={{ fontWeight: 'bold' }}>Confirm WP Completed</Typography>
-        </Grid>
-        <Grid item xs={9}>
-          <Typography>{booleanPipe(cr.confirmDone)}</Typography>
-        </Grid>
-      </Grid>
-    </PageBlock>
+    <InfoBlock title={'Confirm WP Completed'} children={<Typography>{booleanPipe(cr.confirmDone)}</Typography>}></InfoBlock>
```

Comment:
nice love this abstraction. Same comment ab moving the content out of the children prop, but otherwise lit